.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "workbench.colorCustomizations": {
+    "tab.unfocusedActiveBorder": "#fff0"
+  }
+}
```

Comment:
Woops, this accidentally got committed! Mostly commenting to make sure you know that this got sucked into the project folder, in case you need it for your own personal setup